components/nurseries/Summary.vue

```diff
@@ -66,9 +66,23 @@ export default {
 
   computed: {
     businessHours() {
+      let openingTime;
+      let closingTime;
+      
+      if(this.item.nursery)
+      {
+        openingTime = this.item.nursery.facility.openingTime;
+        closingTime = this.item.nursery.facility.closingTime;
+      }
```

Comment:
フォーマッタが整備できていないですが、こんな感じがよいです(ちなみに、セミコロンなくてもよいです)。

```suggestion
      if (this.item.nursery) {
        openingTime = this.item.nursery.facility.openingTime;
        closingTime = this.item.nursery.facility.closingTime;
      }
```